aten/src/ATen/detail/ORTHooksInterface.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <c10/util/Exception.h>
+#include <c10/util/Registry.h>
+
+constexpr const char* ORT_HELP = " ONNX Runtime (ORT) support loads dynamically via 'import torch_ort'.";
```

Comment:
Would this be helpful if it was more guided: "You need to 'import torch_ort' to be able to use the 'ort' device in pytorch" ?